### ISSUE TITLE
Initial issue with the get_feeders call

### ISSUE THREAD

**Issue by gonenutsbrb** [Techzune/petsafe-smartfeed#5]
Hey there, just getting back around to messing with this after the API change, so I'll see if I can't fix it on my fork, but I was get this right away when running the initial get feeders call:

```
>>> devices = sf.devices.get_feeders(client)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.10/site-packages/petsafe_smartfeed/devices.py", line 12, in get_feeders
    response = client.sf_get("feeders")
  File "/usr/lib/python3.10/site-packages/petsafe_smartfeed/client.py", line 131, in sf_get
    return requests.get(URL_SF_API + path, headers=self.get_headers())
  File "/usr/lib/python3.10/site-packages/petsafe_smartfeed/client.py", line 101, in get_headers
    if time.time() >= self.token_expires_time - 10:
TypeError: unsupported operand type(s) for -: 'NoneType' and 'int'
```

Looks like when the token_expires_time is left at 'None' it can't make the subtraction and fails.

Thanks for all the work on the new API!

**Comment by gonenutsbrb** [comment]
Ah I think I'm figuring it out.

I didn't initialize the 'client' with that python session (I had already done the email code process earlier), I just dropped the tokens into the first PetSafeClient call, and it was therefore missing the `token_expires_time` value and just left 'None' in.  Should probably have a catch for that as well though just in case someone like me does something foolish.

**Comment by Techzune** [comment]
Hey @gonenutsbrb,
AWESOME idea!! There are plenty of sections that could use some error handling that I have not implemented.
I'll drop a commit/fix in for this.

**Comment by Techzune** [comment]
Fixed in version 2.2.
The API now defaults `token_expires_time` as `0` -- which will force `headers` to call `refresh_tokens`. 😄
